src/monocle_apptrace/wrap_common.py

```diff
@@ -17,7 +17,9 @@
 TAGS = "tags"
 SESSION_PROPERTIES_KEY = "session"
 INFRA_SERVICE_KEY = "infra_service_name"
-
+TYPE = "type"
+PROVIDER = "provider"
```

Comment:
for model we are using `provider_name` property, maybe we could use same for vector store 
cc @oi-raanne

Reply:
+1